UnityProject/Assets/Scripts/Actions/ActionData.cs

```diff
@@ -100,7 +100,7 @@ public ActionButtonData ToActionButtonData()
 			Type = ActionType == UIActionType.Toggle ? Actions.V2.ActionType.Activated : Actions.V2.ActionType.Trigger,
 			AnimatedIconCatalogue = Sprites,
 			HasCustomCursor = HasCustomCursor,
-			CursorTexture = CursorTexture,
```

Comment:
ok, Technically this is a lazy way of doing it, you're supposed update CursorTexture in the SOs too, that's what I meant 
<img width="967" height="105" alt="image" src="https://github.com/user-attachments/assets/c3af5b6b-0944-4a8f-8432-d6f0c8a0fc83" />
There's a bunch of overrides like this

Reply:
Bod
This doesn't touch SOs at all
The `ToActionButtonData()` function is a function to convert old v1 stuff to v2 stuff, but its not being used anywhere at the moment.

Reply:
ok cool